backend/src/main/java/com/redhat/cloud/notifications/routers/NotificationResource.java

```diff
@@ -477,4 +482,44 @@ public List<BehaviorGroup> findBehaviorGroupsByBundleId(@Context SecurityContext
         );
         return behaviorGroups;
     }
+
+
+    @GET
+    @Path("/eventTypes/{eventTypeId}/endpoints")
+    @Produces(APPLICATION_JSON)
+    @Operation(summary = "Retrieve the endpoints linked to an event type.")
+    @RolesAllowed(ConsoleIdentityProvider.RBAC_READ_NOTIFICATIONS)
+    public Page<Endpoint> getLinkedEndpoints(@Context SecurityContext sec, @PathParam("eventTypeId") UUID eventTypeId, @BeanParam @Valid Query query, @Context UriInfo uriInfo) {
+        String orgId = getOrgId(sec);
+
+        final List<Endpoint> endpoints = endpointEventTypeRepository.findEndpointsByEventTypeId(orgId, eventTypeId, query);
+
+        return new Page<>(
+            endpoints,
+            PageLinksBuilder.build(uriInfo.getPath(), endpoints.size(), query),
+            new Meta(Long.valueOf(endpoints.size()))
+        );
+    }
```

Comment:
I think we should return a "404" if the specified event type does not exist.

Reply:
Sure